packages/neuron-ui/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_NETWORKS=[{"name":"Testnet","remote":"http://localhost:8114"},{"name":"Localhost","remote":"http://localhost:8114"}]
```

Comment:
Shouldn't this belong to the `neuron-wallet` module?

And it should not be `ENV`, as we're only to have official testnet/mainnet in the future. This should be in program constants.